tests/test_Consumer.py

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env python
 
-from confluent_kafka import Consumer, TopicPartition, KafkaError, KafkaException, TIMESTAMP_NOT_AVAILABLE, OFFSET_INVALID, libversion
+from confluent_kafka import (Consumer, TopicPartition, KafkaError, KafkaException, TIMESTAMP_NOT_AVAILABLE,
+                             OFFSET_INVALID, libversion)
```

Comment:
..but split over multiple lines here?
This inconsistency is at least consistent with Python's consistent inconsistency